benchmark.go

```diff
@@ -280,6 +297,7 @@ func outputPlaintext(out output, latencyValues map[float64]time.Duration, summar
 	out.Printf("Elapsed time (seconds):   %.2f\n", summary.ElapsedTimeSeconds)
 	out.Printf("Total requests:           %v\n", summary.TotalRequests)
 	out.Printf("RPS:                      %.2f\n", summary.RPS)
+	// TODO: add sent/received stream messages summary
```

Comment:
Can elaborate why this was not done in this PR? This sounds like the critical feature for benchmark no?

Reply:
It isn't critical in terms of functioning, but it does have some additional information such as total number of stream messages sent/received. The reason of not working on the change in this PR was to avoid any I/O changes apart from core stream benchmarking. This will be worked upon in a followup PR